ord_schema/message_helpers.py

```diff
@@ -739,7 +740,8 @@ def fetch_dataset(dataset_id: str, timeout: float = 10.0) -> dataset_pb2.Dataset
         RuntimeError: If the request fails.
         ValueError: If the dataset ID is invalid.
     """
-    from ord_schema import validations  # Avoid circular import; pylint: disable=import-outside-toplevel.
+    # Avoid circular import; pylint: disable=import-outside-toplevel.
```

Comment:
this comment needs to stay on the same line for the pylint flag to be localized

Reply:
Resolved in [d77d77a](https://github.com/open-reaction-database/ord-schema/pull/701/commits/d77d77a3a92e8dbc3b8d17b6e8633ed6a9d4e7f8).